cernopendata/modules/fixtures/data/docs/cms-about/cms-about.md

```diff
@@ -65,7 +65,8 @@ The following are provided through this portal:
     * The release of 2010 data is accompanied by a small set of simulated data.
     * The release of 2011 data includes some simulated data, limited to those datasets that were reprocessed with a software release compatible with the 2012 collision data.
     * The release of 2012 data includes a larger sample of simulated data. A part of 2012 simulated data is released with the bibliographic information content only, and these datasets will be made available online on demand.
-    * The release of 2015 includes a large collection of simulated data, reprocessed with a software release compatible with the 2015 collision data, but it may still happen that some simulated data did not make it to this reprocessing and are therefore not available in this collection.
+    * The release of heavy-ion-related data of 2013 includes simulated data corresponding to different collision types and centre-of mass energies.
```

Comment:
Cosmetics: could we say something like:

- The release of 2013 heavy-ion-related data includes...

Just for the visual consistency of always starting the list items with YYYY...

Reply:
Yes, that's better. Updated.